desc/compute/_surface.py

```diff
@@ -47,13 +46,10 @@ def _x_FourierRZToroidalSurface(params, transforms, profiles, data, **kwargs):
     coordinates="rtz",
     data=["x"],
     parameterization="desc.geometry.core.Surface",
-    basis="{'rpz', 'xyz'}: Basis for returned vectors, Default 'rpz'",
 )
 def _X_Surface(params, transforms, profiles, data, **kwargs):
     coords = data["x"]
-    if kwargs.get("basis", "rpz").lower() == "rpz":
-        # if basis is rpz, then "x" is rpz and we must convert to xyz
-        coords = rpz2xyz(coords)
+    coords = rpz2xyz(coords)
```

Comment:
Why are we converting this to Cartesian coordinates? I thought the whole point of this PR is to return everything in $(R,\phi,Z)$ coordinates?

Reply:
I thought it wouldn't make sense that `X` returns `R`, or `Y` returns `Phi`. We already have `R`, `Phi` and `Z` for `rpz` coordinates.

Reply:
X, Y, R and Phi should be the only data type that have strict basis definition, if I remember correctly

Reply:
Oh I see now. Yes this makes sense